client/X11/xf_cliprdr.c

```diff
@@ -1099,6 +1127,7 @@ static UINT xf_cliprdr_send_client_format_list(xfClipboard* clipboard)
 	}
 
 	formatList.msgFlags = CB_RESPONSE_OK;
+	formatList.msgType = CB_FORMAT_LIST;
```

Comment:
duplicate, look 3 lines below

Reply:
Oh yes, your cmmit cbee45aace1bb4fd485633224db5433443a201fe has obsoleted one of mine.